src/MarketPlace.sol

```diff
@@ -241,6 +243,11 @@ contract MarketPlace {
         return true;
     }
 
+    function setIllumAdapter(address i) external authorized(admin) returns (bool) {
```

Comment:
Also here, I don't think we need to abbreviate `illuminateAdapter` to `illumAdapter`...

Reply:
> now we have the illuminate adapter being set through a setter method, while the rest of the adapters are still being set through the createMarket method, resulting in the adapters and tokens arrays being of different length. To me this feels incohesive, and most importantly, unexpected.

I dont believe this is the case. This PR actually ensures they are the same length on market create.

The point of the change is that:
1. You cannot know the illuminate PT address before market create
2. Knowing the above, you will ALWAYS have different lengths between adapter and token array length, unless you have a setter for the illuminate adapter that has already been created.
3. If you utilize `illuminateAdapter` and the return from `creator.create()` as `adapters[0]` and `tokens[0]` respectively, then the inputs to market create are consistently populating the market in the index after `0`.

Reply:
That said -- 

> I feel like, a more intentional design around the adapters could be to have a setAdapters (address[] a) method on the MarketPlace, which will set all the adapters on their appropriate index and store the adapters array on the contract. Then we skip passing in the adapters in the createMarket altogether, instead copying the adapters array of the MarketPlace. So you only pass in the principal tokens to the createMarket method.

This also sounds reasonable -- it does add costs on our end (e.g. changing a single adapter means storing the entire array each time), but that should be relatively negligible ($2-30) each time we call it.